README.md

```diff
@@ -51,10 +51,22 @@ Once activated, you will see a video icon in the channel header. Clicking the ic
 
 ![image](https://user-images.githubusercontent.com/177788/42196048-af54d2b8-7e30-11e8-80a0-5e160ae06f03.png)
 
+Optional Webhook Configuration
+---
+
+After a call has completed, Zoom can send a webhook to Mattermost with some basic information about the call (Success, Date, Duration) and this will update the meeting link within Mattermost. 
+
+![image](https://user-images.githubusercontent.com/915956/66890114-c5424480-ef99-11e9-87b9-8e1f6ca75474.png)
+
+Use this webhook URL format to send a webhook when a [Meeting has Ended](https://marketplace.zoom.us/docs/api-reference/webhook-reference/meeting-events/meeting-ending) over to Mattermost:  `https://SITEURL/plugins/zoom/webhook?secret=WEBHOOKSECRET` 
```

Comment:
Yah, this may need some more detail. Do you have access to an admin Zoom account? Maybe we should bug someone like Camille to see if we can have the password and login for an admin on our own MM zoom account, and  then you can test setting it up on a local dev server.  (Or if you give me the credentials, I can do the setup on my machine and I'll screencap the steps for this section.)

Reply:
Don't have access to an admin account, will ping Hanna.

Reply:
So, Good News: Got access to our admin account.  Bad news: Zoom has moved webhooks to their marketplace.  You now need to create an APP so that you can get a webhook to go out. Yes, each of our clients now needs to create an unpublished app on their marketplace. I will need to document this a bit with some screens.

Reply:
Another vendor that is only geared well to support single-instance SaaS integrations. (Jira Cloud, Giphy, potentially Microsoft Office...)